utils/githooks/pre-commit.d/70-isort.sh

```diff
@@ -10,6 +10,13 @@ set -ue
 
 _print_githook_header "isort"
 
+py_files=$(_git_diff_cached_files "*.py SConstruct *SConscript")
```

Comment:
what does `*SConscript` mean here? Is that matching subdirs or filenames?  All files should be called `SConscript` that I'm aware of.

Reply:
It's to match subdirs. Maybe I should do `*/SConscript`?